src/RawModelExportItem.tsx

```diff
@@ -0,0 +1,68 @@
+import Divider from '@mui/material/Divider'
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import Box from '@mui/system/Box'
+import Button from '@mui/material/Button'
+
+import { useState } from 'react'
+import { useRouter } from 'next/router'
+import DirectoryTreeView from './common/DirectoryTreeView'
+import SplitButton from './common/SplitButton'
+
+function RawModelExportItem({ deploymentUuid, version }: { deploymentUuid: string; version: any }) {
+  const router = useRouter()
+  const [displayTree, setDisplayTree] = useState<boolean>(false)
+  const [codeMenuItems, setCodeMenuItems] = useState<string[]>(['Show Code File Tree'])
+
+  const handleCodeMenuItemClicked = (item: string) => {
+    if (item === 'Show Code File Tree') {
+      setDisplayTree(!displayTree)
```

Comment:
This should be "true". Line #23 should be "false". I'm not 100% convinced of this implementation though and think it's something we could revisit

Reply:
That was a holdover from before there was an if statement